<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>My PySide6 App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblTitle">
      <property name="text">
       <string>Hello from Qt Designer</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtName">
      <property name="placeholderText">
       <string>Enter your nameâ€¦</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnGreet">
      <property name="text">
       <string>Greet Me</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblResult">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>

